modules/custom/az_event/config/install/core.entity_form_display.node.az_event.default.yml

```diff
@@ -89,9 +89,15 @@ content:
     weight: 11
     settings:
       show_extra: false
+      hide_date: false
```

Comment:
I've added configuration to not hide the end date field on smart date, but the field is still hidden anyway. This may be a smart date issue. I may look for a patch or resolution to this in a followup.

Reply:
> I've added configuration to not hide the end date field on smart date, but the field is still hidden anyway. This may be a smart date issue. I may look for a patch or resolution to this in a followup.

Sounds like it'd be a good thing to create a Quickstart issue for, at least.